openhands/events/action/commands.py

```diff
@@ -29,6 +29,7 @@ class CmdRunAction(Action):
     runnable: ClassVar[bool] = True
     is_confirmed: ActionConfirmationStatus = ActionConfirmationStatus.CONFIRMED
     security_risk: ActionSecurityRisk | None = None
+    timeout: int | None = None
```

Comment:
We actually have a `.timeout` attribute for all `Event`:

https://github.com/All-Hands-AI/OpenHands/blob/b78f646b656a325ef8e2e185bbb9e67263d7750f/openhands/events/event.py#L47-L51

Which is automatically setted here:

https://github.com/All-Hands-AI/OpenHands/blob/b78f646b656a325ef8e2e185bbb9e67263d7750f/openhands/runtime/runtime.py#L112-L122

This allow us to set a timeout for all the events without have to repeatedly add `.timeout` attribute for all actions 🥵 

I think maybe we can use this directly, without actually need that `--cmd-timeout` which is another more config to care about?

Reply:
Oh, the `.timeout` should be passed from the backend to the client in `on_event`